Build failure on OS X Sierra

I tried to build Remacs on a Mac yesterday, and make fails:

```
RUSTFLAGS= \
	cargo build  --manifest-path=../rust_src/remacs-lib/Cargo.toml
   Compiling lazy_static v0.2.10
   Compiling libc v0.2.21
   Compiling ident_case v1.0.0
   Compiling unicode-xid v0.0.4
   Compiling quote v0.3.15
   Compiling synom v0.11.3
   Compiling errno v0.2.3
   Compiling rand v0.3.15
   Compiling syn v0.11.11
   Compiling darling_core v0.2.0
   Compiling darling_macro v0.2.0
   Compiling darling v0.2.0
   Compiling remacs-util v0.1.0 (file:///Users/whughes/personal/remacs/rust_src/remacs-util)
   Compiling remacs-lib v0.1.0 (file:///Users/whughes/personal/remacs/rust_src/remacs-lib)
    Finished dev [unoptimized + debuginfo] target(s) in 7.74 secs
  CCLD     etags
ld: warning: directory not found for option '-L.././rust_src/target/debug'
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/whughes/personal/remacs/lib-src/../rust_src/remacs-lib/target/debug/libremacs_lib.a(std-9fb646d603da105f.std1-f2fa0d3dcbd22e0b15d25c62d94852f2.rs.rcgu.o):
warning: could not find referenced DIE
clang: error: unable to execute command: Segmentation fault: 11
clang: error: dsymutil command failed due to signal (use -v to see invocation)
make[1]: *** [etags] Error 254
make: *** [lib-src] Error 2
```

It looks like build.rs is failing to find globals.h, but I'm surprised I only get this traceback when I run `cargo build' directly:

```
$ cd rust_src
$ RUST_BACKTRACE=1 cargo build
   Compiling quote v0.3.15
   Compiling md5 v0.3.6
   Compiling nodrop v0.1.12
   Compiling typenum v1.9.0
   Compiling libc v0.2.33
   Compiling ident_case v1.0.0
   Compiling byte-tools v0.1.3
   Compiling sha1 v0.2.0
   Compiling fake-simd v0.1.2
   Compiling lazy_static v0.2.11
   Compiling unicode-xid v0.0.4
   Compiling byteorder v1.1.0
   Compiling safemem v0.2.0
   Compiling synom v0.11.3
   Compiling errno v0.2.3
   Compiling remacs v0.1.0 (file:///Users/whughes/personal/remacs/rust_src)
   Compiling remacs-sys v0.1.0 (file:///Users/whughes/personal/remacs/rust_src/remacs-sys)
   Compiling rand v0.3.18
   Compiling alloc_unexecmacosx v0.1.0 (file:///Users/whughes/personal/remacs/rust_src/alloc_unexecmacosx)
   Compiling base64 v0.8.0
   Compiling syn v0.11.11
error: failed to run custom build command for `remacs-sys v0.1.0 (file:///Users/whughes/personal/remacs/rust_src/remacs-sys)`
process didn't exit successfully: `/Users/whughes/personal/remacs/rust_src/target/debug/build/remacs-sys-36b537ec53d1d6d4/build-script-build` (exit code: 101)
--- stderr
thread 'main' panicked at 'Failed to open globals file: Error { repr: Os { code: 2, message: "No such file or directory" } }', src/libcore/result.rs:906:4
stack backtrace:
   0: std::sys::imp::backtrace::tracing::imp::unwind_backtrace
             at src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
   1: std::sys_common::backtrace::_print
             at src/libstd/sys_common/backtrace.rs:68
   2: std::panicking::default_hook::{{closure}}
             at src/libstd/sys_common/backtrace.rs:57
             at src/libstd/panicking.rs:381
   3: std::panicking::default_hook
             at src/libstd/panicking.rs:397
   4: std::panicking::begin_panic
             at src/libstd/panicking.rs:577
   5: std::panicking::begin_panic
             at src/libstd/panicking.rs:538
   6: std::panicking::try::do_call
             at src/libstd/panicking.rs:522
   7: std::panicking::try::do_call
             at src/libstd/panicking.rs:498
   8: <core::ops::range::Range<Idx> as core::fmt::Debug>::fmt
             at src/libcore/panicking.rs:71
   9: core::result::unwrap_failed
             at /Users/travis/build/rust-lang/rust/src/libcore/macros.rs:23
  10: <core::result::Result<T, E>>::expect
             at /Users/travis/build/rust-lang/rust/src/libcore/result.rs:799
  11: build_script_build::generate_globals
             at remacs-sys/build.rs:138
  12: build_script_build::main
             at remacs-sys/build.rs:201
  13: panic_unwind::dwarf::eh::read_encoded_pointer
             at src/libpanic_unwind/lib.rs:99
  14: rust_panic
             at src/libstd/panicking.rs:459
             at src/libstd/panic.rs:365
             at src/libstd/rt.rs:58
  15: build_script_build::main

warning: build failed, waiting for other jobs to finish...
error: build failed
```

There isn't a `src/globals.h` in my tree -- perhaps we're missing a dependency in our makefile? It's weird that the Travis build is happy.

Same error here after running `make clean` and then `make`

```
ld: warning: directory not found for option '-L.././rust_src/target/release'
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std1-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std14-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std14-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std14-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std14-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std14-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std14-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std4-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std4-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std4-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
while processing /Users/christianjohansson/Documents/Remacs/lib-src/../rust_src/remacs-lib/target/release/libremacs_lib.a(std-24169c95316ab127.std4-7120ec8fbd47f610cea3c5bef5e09005.rs.rcgu.o):
warning: could not find referenced DIE
clang: error: unable to execute command: Segmentation fault: 11
clang: error: dsymutil command failed due to signal (use -v to see invocation)
make[1]: *** [etags] Error 254
make: *** [lib-src] Error 2
```

--- a/src/Makefile.in
+++ b/src/Makefile.in
@@ -606,7 +606,7 @@ macosx_alloc_dir=$(rust_srcdir)/alloc_unexecmacosx
 MACOSX_ALLOC_INPUTS=$(macosx_alloc_dir)/src/** $(macosx_alloc_dir)/Cargo.*
 cargo_manifest=$(rust_srcdir)/Cargo.toml
 LIBREMACS_ARCHIVE=$(rust_srcdir)/target/$(CARGO_BUILD_DIR)/$(REMACSLIB_NAME)
-$(LIBREMACS_ARCHIVE): $(rust_srcdir)/src/** $(rust_srcdir)/remacs-sys/*.rs $(rust_srcdir)/remacs-sys/Cargo.* $(rust_srcdir)/Cargo.* $(MACOSX_ALLOC_INPUTS)
+$(LIBREMACS_ARCHIVE): globals.h $(rust_srcdir)/src/** $(rust_srcdir)/remacs-sys/*.rs $(rust_srcdir)/remacs-sys/Cargo.* $(rust_srcdir)/Cargo.* $(MACOSX_ALLOC_INPUTS)
 	RUSTFLAGS=$(RUSTFLAGS) \
 	$(CARGO_BUILD) $(CARGO_FLAGS) --manifest-path $(cargo_manifest)
 

